src/test/apps.test.js

```diff
@@ -555,3 +559,62 @@ describe("App validator", function () {
     expect(errors[2].attribute).to.equal("disk");
   });
 });
+
+describe("App Page component", function () {
+
+  beforeEach(function () {
+    var app = _.extend(appScheme, {
+      id: "/test-app-1",
+      healthChecks: [{path: "/", protocol: "HTTP"}],
+      tasksStaged: 0,
+      tasksRunning: 1,
+      tasksHealthy: 0,
+      tasksUnhealthy: 1,
+      status: AppStatus.RUNNING,
+      tasks: [
+        {
+          id: "test-task-1",
+          host: "127.0.0.1",
+          ports: [31857],
+          startedAt: "2015-07-07T09:01:11.689Z",
+          stagedAt: "2015-07-07T09:01:11.130Z",
+          version: "2015-07-06T15:13:21.875Z",
+          appId: "/test-app-1",
+          healthStatus: HealthStatus.UNHEALTHY,
+          healthCheckResults: [
+            {
+              alive: false,
+              consecutiveFailures: 3,
+              firstSuccess: "2015-07-07T09:15:31.752Z",
+              lastFailure: "2015-07-07T09:18:08.943Z",
+              lastSuccess: "2015-07-07T09:17:52.306Z",
+              taskId: "test-task-1"
+            }
+          ]
+        }
+      ]
+    });
+
+    AppsStore.apps = [app];
+
+    this.renderer = TestUtils.createRenderer();
+    this.renderer.render(<AppPageComponent appId={app.id} router={{}} />);
+    this.component = this.renderer.getRenderOutput();
+    this.element = this.renderer._instance._instance;
+  });
+
+  afterEach(function () {
+    this.renderer.unmount();
+  });
+
+  it("has the correct app id", function () {
+    var appId = this.component.props.children[0].props.appId;
+    expect(appId).to.equal("/test-app-1");
+  });
+
+  it("returns the right health message for failing tasks", function () {
```

Comment:
Could you add tests also for HealthStatus.HEALTHY and HealthStatus.UNKNOWN ?
Thx!